pkg/controller/migmigration/backup.go

```diff
@@ -265,7 +265,6 @@ func (t *Task) hasBackupCompleted(backup *velero.Backup) (bool, []string) {
 			"Backup: %s/%s failed.",
 			backup.Namespace,
 			backup.Name)
-		reasons = append(reasons, message)
```

Comment:
This change should go into BackupPhasePartiallyFailed rather than BackupPhaseFailed